change/@ni-nimble-components-573fc816-eaa7-4e07-b0ef-8bfd5ccff19e.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
Think it'd be worthwhile to add some developer docs reminding us why/how we should do this for future components? I could imagine it going in spec templates or in nimble-components/CONTRIBUTING.md. It's kinda sorta similar to [this section of the latter](https://github.com/ni/nimble/blob/main/packages/nimble-components/CONTRIBUTING.md#focus-delegation) so that could be a good place to put it, but I don't feel too strongly.

Reply:
I added it to that section of `CONTRIBUTING.md`.